documentation/inventory_file_v2.md

```diff
@@ -68,18 +68,14 @@ groups:
 
 ### Target object
 
-You can specify a target with the string representation of a URI, or as a hash
-with any of the following fields:
+Specify a target under the `targets` key. You can specify a target's name in the following ways:
+- Use an array with the string representation of the target's Universal Resource Identifier (URI).
```

Comment:
```suggestion
- Use the string representation of the target's Universal Resource Identifier (URI).
```

The `targets` key accepts an array where each element is either a string or hash.